sale_order_move_menu/views/stock_move_views.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="stock_move_move_menu_search" model="ir.ui.view">
+        <field name="name">stock.move.search (Sale Moves)</field>
+        <field name="model">stock.move</field>
+        <field name="arch" type="xml">
```

Comment:
We'll fix here a potential error that comes from v13 implementation. This view is completely new, but linked to an existing model that already has a `search` view. For these situations, in order to prevent that this view is accidentally selected by Odoo at other stock move actions, it's better to set a low `priority` (like e.g. `99`)